dist/components/import.html

```diff
@@ -0,0 +1,2 @@
+<link rel="import" href="/vendor/polymer/polymer.html">
+<link rel="import" href="/components/top-bar/top-bar.html">
```

Comment:
wondering if it should just be deleted after the import-build file is created then.